lighthouse-core/test/lib/stack-packs-test.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const lighthouseStackPacks = require('lighthouse-stack-packs');
```

Comment:
I saw your comment in https://github.com/GoogleChrome/lighthouse-stack-packs/pull/44#issuecomment-689105854. To add to that,

https://github.com/GoogleChrome/lighthouse-stack-packs/blob/409530a6712ffe1f7e57d9db00c675469c99117c/index.js#L1-L8

also needs to export `joomla.js` for it to show up in lighthouse reports.

Reply:
good catch, just did that.